v1.1.1/js/Loopy.js

```diff
@@ -77,6 +82,9 @@ function Loopy(config){
 		if(!self.modal.isShowing){ // modAl
 			self.model.update(); // modEl
 		}
+		window.addEventListener("wheel", self.zoom);
+		window.addEventListener("keydown", self.buttonZoom);
+		window.addEventListener("keydown", self.moveWindow);
```

Comment:
Currently, these event listener registrations are happening within an update function. While this doesn't end up registering the same event more than once, it is probably better to have a single registration somewhere that runs once.

Additionally, there is the possibility of using the "subscribe" method from minpubsub to match other code within the project.